core/mobile/src/backend/MobileRpcServer.ts

```diff
@@ -170,7 +195,7 @@ export function setupMobileRpc() {
       return;
     }
 
-    server = new MobileRpcServer();
+    server = new MobileRpcServer(pendingMessages);
```

Comment:
A basic solution to the error throwing problem would be to do the following:

```ts
try {
  server = new MobileRpcServer(pendingMessages);
} catch (_error) {
  server = new MobileRpcServer();
}
```
That way, at least having an error happen while sending the pending messages wouldn't break everything.

Reply:
Given that the error happens in a callback, it seems that it won't get thrown, so this change won't do anything. Sorry for the confusion.

Reply:
See my response to the related comment.